js/unison.js

```diff
@@ -1,5 +1,15 @@
-Unison = (function() {
-
+/*jshint indent:2 */
+/*global define */
+(function (root, factory) {
+  'use strict';
+  if (typeof define === 'function' && define.amd) {
+    define([], function () {
+      return (root.Unison = factory());
+    });
+  } else {
+    root.Unison = factory();
+  }
```

Comment:
Looks like this is only AMD? Since we're adding a wrapper here, might be able to use UMD to also support CommonJS/Node-style: https://github.com/umdjs/umd